src/Jobs/JobType.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types = 1);
+
+namespace BE\QueueManagement\Jobs;
+
+use MabeEnum\Enum;
+
+/**
+ * @final
+ */
+class JobType extends Enum
+{
+    public const SQS = 'sqs';
+    public const RABBIT_MQ = 'rabbitMq';
```

Comment:
```suggestion
    public const RABBIT_MQ = 'RabbitMQ';
```

...když můžem, tak bych použil oficiální kapitalizaci...